docs/application/toc_all.md

```diff
@@ -83,6 +83,7 @@
 #### Components
 ##### [Overview](/application/dotnet/guides/nui/ui-components.md)
 ##### [Styling UI Components](/application/dotnet/guides/nui/styling-controls-with-JSON.md)
+##### [Themes and Xaml](/application/dotnet/guides/nui/styling-controls-with-xaml.md)
```

Comment:
I would suggest to please look for a more relevant name for toc.

Reply:
How about? "Styling UI Compoenents with Xaml"

Reply:
Yes seems fine @mihashco

Reply:
@mihashco can you please update it?

Reply:
Sorry for late reply. Suggestions was updated. Thank you!